pilot/pkg/xds/adapter.go

```diff
@@ -62,20 +62,28 @@ func UpgradeV2Request(v2Req *xdsapi.DiscoveryRequest) *discovery.DiscoveryReques
 	}
 }
 
-// Convert from v3 to v2
+// DowngradeV3Response converts from v3 to v2
```

Comment:
FYI: I have a PR in progress to change adsc to use v3 by default. 

We should stop using v2 as much as possible.